src/ui/server/controllers/insurance/your-buyer/your-buyer-details/index.ts

```diff
@@ -0,0 +1,52 @@
+import { PAGES } from '../../../../content-strings';
+import { Request, Response } from '../../../../../types';
+import { mapCountries } from '../../../../helpers/mappings/map-countries';
+import api from '../../../../api';
+import { FIELD_IDS, ROUTES, TEMPLATES } from '../../../../constants';
+import singleInputPageVariables from '../../../../helpers/page-variables/single-input/insurance';
+
+const FIELD_ID = FIELD_IDS.BUYER_COUNTRY;
+
+export const PAGE_VARIABLES = {
+  FIELD_ID,
+  PAGE_CONTENT_STRINGS: PAGES.YOUR_BUYER,
+};
+
+export const get = async (req: Request, res: Response) => {
+  if (!req.session.submittedData || !req.session.submittedData.insuranceEligibility) {
+    req.session.submittedData = {
+      ...req.session.submittedData,
+      insuranceEligibility: {},
+    };
+  }
```

Comment:
Don't need these checks, this is only required for eligibility routes, not insurance

Reply:
Instead we need to check the application exists - full example here https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/controllers/insurance/policy-and-export/multiple-contract-policy/index.ts#L79